docs/reference/upgrade.asciidoc

```diff
@@ -47,12 +47,15 @@ a|
 
 [WARNING]
 ====
-The following upgrade paths are *not* supported:
+The following upgrade path is *not* supported (both full cluster restart and rolling upgrade):
 
 * 6.8 to 7.0.
-* 6.7 to 7.1.–{version}.
 
-In-place downgrades to earlier versions are not supported. To downgrade to an
+The following upgrade path is supported but requires a <<restart-upgrade, Full cluster restart upgrade>>.
```

Comment:
I don't think this bit should be part of this `WARNING`. I think it's adequately covered by other text on this page.